registration/tests/serializers/test_field_list.py

```diff
@@ -0,0 +1,81 @@
+from django.core.exceptions import ValidationError
+from django.test.testcases import TestCase
+from unittest.mock import patch
+
+from accounts.models import User
+from registration.models import Student, Field
+from registration.serializers import FieldSerializer, FieldListSerializer
+
+
+class StudentSerializerTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create(email="user@staff.com")
+        self.parent_field = Field.objects.create(
+            role=Field.PARENT,
+            name="email",
+            question="What is your email?",
+            question_type=Field.TEXT,
+            is_default=True,
+            order=1,
+        )
+        self.other_parent_field = Field.objects.create(
+            role=Field.PARENT,
+            name="email",
+            question="What is your email?",
+            question_type=Field.TEXT,
+            is_default=True,
+            order=2,
+        )
+        self.child_field = Field.objects.create(
+            role=Field.CHILD,
+            name="Date Of Birth",
+            question="When were you born?",
+            question_type=Field.TEXT,
+            is_default=True,
+            order=1,
+        )
+        self.guest_field = Field.objects.create(
+            role=Field.GUEST,
+            name="Relation to Family",
+            question="How are you related?",
+            question_type=Field.TEXT,
+            is_default=True,
+            order=1,
+        )
+
+    def test_field_list_serializer(self):
+        serializer = FieldListSerializer(child=FieldSerializer(), data=Field.objects)
```

Comment:
kinda confused why the first parameter is called `child`

Reply:
nope, this is a [DRF thing](https://www.django-rest-framework.org/api-guide/serializers/#listserializer) - the FieldSerializer is a child of the FieldListSerializer!

i think we're just really good at naming our models python keywords 😅

Reply:
yeah that's what I meant lol. Thanks for clarifying! That's a kewl feature